.travis.yml

```diff
@@ -32,7 +39,7 @@ deploy:
     skip_cleanup: true
     local-dir: "pkgs/releases"
     on:
-      branch: "redhat-citus"
+      branch: "all-citus"
```

Comment:
Please don't forget to update word doc to mention now we can release os packages using "all" parameter and at once